nautobot_ssot_infoblox/utils/client.py

```diff
@@ -251,27 +279,65 @@ def get_all_ipv4address_networks(self, prefixes):
             }
         ]
         """
+
+        def get_ipaddrs(url_path, data):
```

Comment:
Do you want to add type hints here? Looks like you did on the other functions.

Reply:
Thoughts on docstrings here? The functions are named really well but I think having a docstring there for consistencies sake wouldn't hurt.

Reply:
You're the man. Thanks.

Reply:
@qduk Added some as requested.